test/unit/worker/convert/test_automatic_pdf_downscaling.py

```diff
@@ -0,0 +1,51 @@
+import os
+import logging
+
+from test.unit.worker.convert.convert_test import ConvertTest
+from workers.convert.convert_pdf import split_merge_pdf
+from utils.object import Object
+
+log = logging.getLogger(__name__)
+
+
+class AutomaticPDFDownscalingTest(ConvertTest):
+
+    max_size_in_mb = 0.4
+
+    def test_no_downscaling_if_below_threshold(self):
+        """Do not apply additional scaling if pdf is below the given size threshold."""
+        pdf_src = f'{self.resource_dir}/files/test.pdf'
+        file_generated = f'{self.working_dir}/data/pdf/merged.pdf'
+
+        max_pdf_size = 0.4
+
+        obj = Object(self.working_dir)
+        params = [{"file": "test.pdf", "range": [1, 20]},
+                  {"file": "test.pdf", "range": [21, 27]}]
+
+        with open(pdf_src, 'rb') as stream:
+            obj.add_stream('test.pdf', 'pdf', stream)
+
+        split_merge_pdf(params, obj.get_representation_dir('pdf'))
+        self.assertTrue(os.path.isfile(file_generated))
+        self.assertTrue(os.path.getsize(file_generated) * 0.000001 > max_pdf_size)
```

Comment:
in wie fern ist das eine gute abschätzung ?

Reply:
Es wäre gut max_pdf_size tatsächlich größer als die pdf zu haben und als param der merge funktion mitgeben.

Reply:
Ja, würde ich auch so sehen, hier würde ein paramter (..., max_size_in_mb=1000000) zum Verständnis beitragen.

Reply:
Ok stimmt, der default ist 150, deswegen fällt die 4xx kb Testdatei halt darunter. Aber dann schreibe ich mal explizit was hin.

Reply:
Gut dass ihr noch mal draufgeguckt habt. Dadurch dass ich den Test jetzt noch einmal umgeschrieben habe (mit explizitem Check der Dateigrößen), ist auch noch aufgefallen dass unsere bisherigen kombinierten PDFs unverhältnismäßig groß waren. 

Im Test "zerschneide" ich ja künstlich das Test.pdf und lasse es wieder rekombinieren. Durch den Test auf die Dateigröße habe ich gerade gesehen, dass das Resultat um einen Faktor 1.7 größer ist als das Original (bei gleichem Inhalt). Deshalb habe ich jetzt die merge Funktion noch einmal etwas umgeschrieben.